packages/botonic-react/src/components/button.jsx

```diff
@@ -65,6 +73,23 @@ export const Button = props => {
       window.open(props.url, props.target || '_blank')
     }
     if (props.onClick) props.onClick()
+    if (disableButtons) {
+      // TODO: Extract this logic
+      const parent = document.getElementById(parentId)
```

Comment:
This will not work if `shadowDOM` is active. We should pass the `host` prop from Webchat down to the rest of the tree so that it can be used in Button and other components. I think passing it through the `WebchatContext.Provider` would be a nice solution.